README.md

```diff
@@ -5,9 +5,8 @@ Google Cloud's container platforms:
  [Cloud Run](https://cloud.google.com/run),
  [GKE](https://cloud.google.com/kubernetes-engine),
  [Anthos](https://cloud.google.com/anthos),
- [App Engine](https://cloud.google.com/appengine),
- [Cloud Functions](https://cloud.google.com/functions),
  and [Compute Engine runing Container-Optimized OS](https://cloud.google.com/container-optimized-os/docs).
+ They are also used as part of the build steps of [App Engine](https://cloud.google.com/appengine) and [Cloud Functions](https://cloud.google.com/functions).
```

Comment:
> They are also used as part of the build steps

The builders perform all the build steps for GAE/GCF. Perhaps "They are also used as the build system for"?

Reply:
Great suggesting. Let me edit my PR